server.js

```diff
@@ -0,0 +1,78 @@
+const Hapi = require('hapi');
+const TheRealYou = require('therealyou');
```

Comment:
Don't you think `therealyou` is unnecessary for the example?

Reply:
Maybe put it helps discovery and provides a working example of how to use it. Many people are running node behind a proxy.

Reply:
There is already an example on the README page. I would go for the smallest possible example. This makes it seem like `therealyou` is necessary for running `ip-whitelist`